check-pulp-msgs.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+#
+# Developer ..........: Waldirio M Pinheiro <waldirio@redhat.com>
+# Created at .........: 05/01/2021 
+# Tested Sat Version .: 6.6, 6.7, 6.8, 6.9
+# Purpose ............: Check the qpid messages coming from/to Pulp.
+#                       There is a BZ that is causing an weird behavior
+#                       and this script help to identify it. Also there
+#                       is a KCS, both below
+# 
+#                       https://access.redhat.com/solutions/5911931 
+#                       https://bugzilla.redhat.com/show_bug.cgi?id=1945534
+# 
+
+SRV_FQDN=$(hostname)
```

Comment:
In most places we use `hostname -f`. In most cases it won't be a difference, but worth to remember.

Reply:
Hi

Consider done my friend! Thank you once again!